hbase-server/src/main/java/org/apache/hadoop/hbase/master/normalizer/RegionNormalizerWorker.java

```diff
@@ -55,6 +56,7 @@
   private final RateLimiter rateLimiter;
 
   private final long[] skippedCount;
+  private final boolean defaultNormalizerValue;
```

Comment:
Good to rename it to `defaultNormalizerTableLevel` or something similar to indicate that this value is going to be used if not overridden by `NORMALIZATION_ENABLED` at table descriptor level. Good to add this as comment also.